docs/en/stack/security/authorization/built-in-roles.asciidoc

```diff
@@ -50,6 +50,9 @@ change between releases.
 NOTE: This role does not provide access to the logstash indices and is not
 suitable for use within a Logstash pipeline.
 
+[[built-in-roles-beats-admin]] `beats_admin` ::
+Grants access to the `.management-beats` index for managing configurations.
```

Comment:
Though this matches the logstash_admin description, my two cents is that it might be clearer to say something like this:
"... index, which contains configuration information for the Beats".